plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -186,8 +186,7 @@ private JdbcTableHandle prepareTableHandleForQuery(JdbcTableHandle table)
                                     table.getRequiredNamedRelation().getRemoteTableName().getSchemaName(),
                                     table.getRequiredNamedRelation().getRemoteTableName().getTableName())),
                     table.getConstraint(),
-                    table.getLimit(),
-                    table.getColumns(),
+                    table.getSortOrder(), table.getLimit(), table.getColumns(),
```

Comment:
fmt

(intellij garbles formatting when you use things like 'change signature' to reorder parameters)